src/omnisharp/protocol.ts

```diff
@@ -286,6 +287,18 @@ export interface ProjectInformationResponse {
     DotNetProject: DotNetProject;
 }
 
+export enum DiagnosticStatus
+{
+    Processing = 0,
```

Comment:
could this be done as union type with literals?

Reply:
It presents enum from omnisharp-roslyn api message. I personally prefer string serialization over number ones but existing apis return enums in number form -> i don't think this is good place to brake that consistency. For example `FileModificationType` on many of file operations.